cmd/snap/cmd_aliases_test.go

```diff
@@ -79,11 +80,12 @@ func (s *SnapSuite) TestAliases(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(rest, DeepEquals, []string{})
 	expectedStdout := "" +
-		"App        Alias       Notes\n" +
-		"bar.dump   bar_dump    enabled\n" +
-		"bar.???    bar_dump.1  undefined,disabled\n" +
-		"foo        foo0        auto\n" +
-		"foo.reset  foo_reset   -\n"
+		"App               Alias        Notes\n" +
+		"bar.dump          bar_dump     manual\n" +
+		"bar.dump          bar_dump.1   unaliased\n" +
+		"bar.safe-restore  bar_restore  manual,auto:restore\n" +
+		"foo               foo0         auto\n" +
+		"foo.reset         foo_reset    manual\n"
```

Comment:
A few suggested tweaks to the format:

- s/App/Command/
- s/unaliased/disabled/
- s/auto:restore/override/
- s/auto/-/  (auto is the default and the vast majority of cases, so might be nice to not call it out)